css-grid_place-elements/css/styles.css

```diff
@@ -0,0 +1,73 @@
+/*
+Use grid to fulfill the following tasks:
+
+[ ] - .element__1 should start at column three and go until column five
+[ ] - .element__2 should start at column five and go until the end
+[ ] - .element__5 should span two columns and two rows
+[ ] - .element__8 should span two rows
+[ ] - .element__fish should be in row one and span the entire grid width
+[ ] - .element__20 should start at row six and span two rows
+
+*/
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: system-ui;
+  background-color: #f7fdff;
+}
+
+.container {
+  display: grid;
+  gap: 20px;
+  grid-template-columns: repeat(5, 1fr 2fr);
+  grid-template-rows: repeat(10, 50px);
+  padding: 10px;
+}
+
+.container__element {
+  padding: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgb(186, 234, 246);
+  border-radius: 3px;
+  font-size: 2rem;
+}
+.element-1 {
+  background-color: #3a67a4;
+  grid-column: 3/5;
+}
+
+.element-2 {
+  background-color: #3a67a4;
+  grid-column: 5 / 11;
+}
+
+.element-5 {
+  background-color: #3a67a4;
+  grid-column: span 2;
+  grid-row: span 2;
+}
+
+.element-8 {
+  background-color: #3a67a4;
+  grid-row: span 2;
+}
+
+.element-fish {
+  background-color: #3a67a4;
+  grid-row-start: 1;
+  grid-column: 1 / span 10;
```

Comment:
You could also use: `grid-column: 1 / -1`

`-1` means got to the end. The good thing is that this works regardless of how many columns you have.